src/Aquarius.ONE.ClientSDK/Operations/OperationCache.cs

```diff
@@ -331,26 +331,29 @@ public async Task<bool> LoadAsync(bool loadChildCaches = false)
             CacheColumns();
             return true;
         }
+
         private async Task<bool> LoadComputations(WorksheetDefinition sourceWorksheetDefinition)
         {
-            if (sourceWorksheetDefinition != null)
+            if (sourceWorksheetDefinition == null) 
+                return true;
+
+            foreach (var sourceColumn in sourceWorksheetDefinition.Columns)
             {
-                for (int x = 0; x < sourceWorksheetDefinition.Columns.Count; x++)
-                {
-                    var sourceColumn = sourceWorksheetDefinition.Columns[x];
+                if (sourceColumn.DataSourceBinding?.DataSource != EnumDataSource.DatasourceComputation ||
+                    string.IsNullOrEmpty(sourceColumn.DataSourceBinding.BindingId)) 

```

Comment:
rather than just checking for null or empty here you could check that it is a valid guid.  Any actual computation will have a valid guid Id.